actions/access_keys/search.go

```diff
@@ -17,21 +17,18 @@ import (
 // @Description	Search access key
 // @Tags		Access-key
 // @Produce		json
-// @Param		page query int false "page"
-// @Param		page_size query int false "page_size"
-// @Param		order_by_field query string false "order_by_field"
-// @Param		sort_direction query string false "sort_direction"
-// @Param		metadata query string false "metadata"
-// @Param		conditions query string false "conditions"
-// @Success		200
+// @Param		SearchRequestParameters body actions.SearchRequestParameters false "Supports targeted resource searches with filters for metadata and custom conditions, plus options for pagination and sorting to streamline data exploration and analysis"
```

Comment:
Got this error when tried execute in swagger. "json: cannot unmarshal string into Go struct field QueryParams.params.page of type int". Is it possible to make page and page_size fields integers by default ?
<img width="1204" alt="fields" src="https://github.com/bitcoin-sv/spv-wallet/assets/150656925/0c4324a5-9a70-4d7a-90a2-ba0f3e510bfb">

Reply:
Unfortunately I do not see a way to do it